pkg/ingester/validate.go

```diff
@@ -21,7 +16,7 @@ const (
 )
 
 // ValidateSample returns an err if the sample is invalid
-func ValidateSample(s *model.Sample) error {
+func ValidateSample(s *model.Sample, maxLabelNameLength, maxLabelValueLength int) error {
```

Comment:
Perhaps we should have a `ValidationConfig` that we embed in the `IngesterConfig`, as I'd also want to add the limits from #578 as a flag.

Reply:
moved these into their own config struct